### PR TITLE
Add a board app and add a plugin to show the current board

### LINKED ISSUE #27: make a board member plugin

### PR DESCRIPTION
**Pull request by lukovdm**
closes #27 #28 

Add an app to manage boards and add a plugin to display the current and all previous boards on a page.

### CODE REVIEW

File: website/boards/models.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+class BoardMembership(models.Model):
+    board = models.ForeignKey(Board, on_delete=models.CASCADE)
+    member = models.ForeignKey(Member, on_delete=models.CASCADE)
+
+    function = models.CharField(max_length=255)
+    email = models.EmailField()
+
+    picture = models.ImageField()
```
Comment (NienkePienke01) [inline website/boards/models.py:60]: There should also be a possibility to not add a picture to a board member

File: website/boards/models.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        {"start": "A boards start cannot fall in another board"}
+                    )
+
+    def __str__(self):
+        return f"Board of {self.start.year}"
+
+
+class BoardMembership(models.Model):
+    board = models.ForeignKey(Board, on_delete=models.CASCADE)
+    member = models.ForeignKey(Member, on_delete=models.CASCADE)
```
Comment (NienkePienke01) [inline website/boards/models.py:55]: This will give an error when the old board member is not part of BF2's anymore, and doesn't have an account

**Review by NienkePienke01 — CHANGES_REQUESTED** [review]

File: website/boards/models.py (hunk 1 above)
Comment (lukovdm) [reply website/boards/models.py]: This should be fixed now

File: website/boards/models.py (hunk 2 above)
Comment (lukovdm) [reply website/boards/models.py]: This too

**Review by NienkePienke01 — APPROVED** [review]
Well done! :)

**Pull request merged**